usersync/cookie.go

```diff
@@ -96,29 +165,11 @@ func (cookie *Cookie) SetOptOut(optOut bool) {
 	cookie.optOut = optOut
 
 	if optOut {
-		cookie.uids = make(map[string]uidWithExpiry)
-	}
-}
-
-// Gets an HTTP cookie containing all the data from this UserSyncMap. This is a snapshot--not a live view.
-func (cookie *Cookie) ToHTTPCookie(ttl time.Duration) *http.Cookie {
-	j, _ := json.Marshal(cookie)
-	b64 := base64.URLEncoding.EncodeToString(j)
-
-	return &http.Cookie{
-		Name:    uidCookieName,
-		Value:   b64,
-		Expires: time.Now().Add(ttl),
-		Path:    "/",
+		cookie.uids = make(map[string]UIDEntry)
 	}
 }
 
 // GetUID Gets this user's ID for the given syncer key.
-// The first returned value is the user's ID.
-// The second returned value is true if we had a value stored, and false if we didn't.
-// The third returned value is true if that value is "active", and false if it's expired.
-//
-// If no value was stored, then the "isActive" return value will be false.
 func (cookie *Cookie) GetUID(key string) (string, bool, bool) {
```

Comment:
I agree with the comment removal. In the comment's absense, maybe this is a case where named return parameters make sense simply to make things more readable
```
171
172   // GetUID Gets this user's ID for the given syncer key.
173 - func (cookie *Cookie) GetUID(key string) (string, bool, bool) {
    + func (cookie *Cookie) GetUID(key string) (uid string, isUIDFound bool, isUIDActive bool) {
174       if cookie != nil {
175           if uid, ok := cookie.uids[key]; ok {
176               return uid.UID, true, time.Now().Before(uid.Expires)
177           }
178       }
179       return "", false, false
180   }
usersync/cookie.go
```